server/routes/testSequelizer.js

```diff
@@ -0,0 +1,17 @@
+const { Project, User, Team } = require("../services/dbConnection");
+
+const user = {name: "Jakub", surname: "Pawłowski", team_id: 2};
```

Comment:
Shouldn't we put test files in separate directories?

Reply:
This file shouldn't be merged.

Reply:
It has been deleted from branch.